app/views/registrations/_contact_info.html.haml

```diff
@@ -2,10 +2,16 @@
   .field
     = f.label :phone
     = f.text_field :phone
+  .field
+    = f.label :receive_text_messages
+    = f.radio_button :receive_sms, true, checked: true
+    Send Me Text Message Notifications
+    = f.radio_button :receive_sms, false, checked: false
```

Comment:
The value of `receive_sms` is saved after submitting the registration form.  However, there seems to be a problem when the value is `false`.  After visiting the registration form, I see that "Send Me Text Message Notifications" is selected.